scripts/scripts.js

```diff
@@ -114,6 +114,23 @@ function buildImageCollageForPicture(picture, caption, buildBlockFunction) {
   return newBlock;
 }
 
+function formatAutoblockedImageCaptionsForColumns(block, enclosingDiv) {
+  const picture = block.querySelector('picture');
+  const caption = block.querySelector('p');
```

Comment:
I'm wondering if we'd better add a judgment that skips this logic when there is no caption.

Reply:
This `format..` function only gets called inside the "autoblocking images with captions" function above, so it shouldn't affect normal images

Reply:
That's good